code/game/objects/structures/signs.dm

```diff
@@ -1,16 +1,17 @@
 /obj/structure/sign
-	icon = 'icons/obj/decals_arrow.dmi'
+	icon = 'icons/obj/decals_arrow.dmi' // a copy of icons/obj/decals.dmi with directional arrows on the sprites, so a mapper knows which way a sign is facing
 	anchored = TRUE
 	opacity = FALSE
 	density = FALSE
 	layer = WALL_OBJ_LAYER
 	var/directional = TRUE //if true init to a given x/y offset on a wall, if not leave floating in space. used for multiple signs on a wall to prevent them all from moving to the same offset and overlapping/becoming unreadable
+	var/unmodified_icon = 'icons/obj/decals.dmi' // the clean version of the sprite, which we replace in initialize when the sign loads in game
 
 /obj/structure/sign/Initialize(mapload)
 	. = ..()
-	icon = 'icons/obj/decals.dmi'
 	if(!directional) //if not directional do not initialize to a x or y offset
 		return
+	icon = unmodified_icon
```

Comment:
this whole thing would probs be better handled the way stuff like platforms does it but eh, good enough

Reply:
Yeah I'd rather do that in a seperate PR